kore/test/Test/Kore/Strategies/Common.hs

```diff
@@ -46,6 +49,7 @@ runVerification stepLimit axioms claims =
     runSimplifier mockEnv
     $ runExceptT
     $ Verification.verify
+        BreadthFirst
```

Comment:
I'm not sure if I should add a `GraphSearchOrder` parameter to `runVerification` or live `BreadthFirst` here

Reply:
No, I think this is fine. The test was written for `BreadthFirst` originally.